src/ResourceManager/Insights/Commands.Insights/Metrics/GetAzureRmMetricCommand.cs

```diff
@@ -16,20 +16,20 @@
 using System.Linq;
 using System.Management.Automation;
 using System.Text;
-using System.Threading;
 using System.Xml;
 using Microsoft.Azure.Commands.Insights.OutputClasses;
 using Microsoft.Azure.Management.Monitor;
 using Microsoft.Azure.Management.Monitor.Models;
 using Microsoft.Rest.Azure.OData;
+using System.Globalization;
 
 namespace Microsoft.Azure.Commands.Insights.Metrics
 {
     /// <summary>
     /// Get the list of metric definition for a resource.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "AzureRmMetric"), OutputType(typeof(PSMetric))]
-    public class GetAzureRmMetricCommand : MonitorClientCmdletBase
+    [Cmdlet(VerbsCommon.Get, "AzureRmMetric"), OutputType(typeof(PSMetric[]))]
```

Comment:
@pkamath13 @gucalder you can keep the output type to `PSMetric` -- for cmdlets that return a list of objects, we still want to have the output type as the concrete (element) type